### DISCUSSION TITLE
how to encrypt ID in edit mode ?

### DISCUSSION THREAD

**Discussion by shedenk** [filamentphp/filament#6003] [Help]
![Screenshot 2023-03-23 at 10 05 17](https://user-images.githubusercontent.com/10915743/227091228-ab91e97d-0166-4b49-9d15-c856aee4604e.png)

**Comment by danharrin** [comment]
Use a UUID primary key or something?

**Reply by shedenk** [reply]
use id then encrypt/decrypt is it possible?

**Reply by danharrin** [reply]
Maybe try using a HashID then?

**Reply by shedenk** [reply]
sample ?

**Reply by danharrin** [reply]
I'm not writing code for you... take a look at https://github.com/vinkla/laravel-hashids

**Comment by shedenk** [comment]
I tried using this guide
but still failed

https://filamentphp.com/tricks/masking-ids-in-urls-using-hashids

url : 'http/127.0.0.1:8001/admin/ads/providers/x7zlo3l5mn/edit'
Error : 404 NOT FOUND
